src/stories/select/simple/appearance.stories.js

```diff
@@ -0,0 +1,88 @@
+import React from "react"
+import { action } from "@storybook/addon-actions"
+import Select from "../../../components/select"
+
+const handleChange = () => action("handleChange")
+
+export default {
+  title: "Select/Appearance",
+  component: Select,
+  args: {
+    storyName: "",
+    options: [
+      { value: 1, name: "One" },
+      { value: 2, name: "Two" },
+      { value: 3, name: "Three" },
+      { value: 4, name: "Four" },
+      { value: 5, name: "Five" },
+      { value: 6, name: "Six" },
+    ],
+    allowCustomValues: false,
+    skipContainer: false,
+    withAutosuggest: false,
+    handleChange: handleChange(),
```

Comment:
here you are calling the fuction `handleChange`, is it correct? Or you want just to assign it?